server/game/cards/05-DT/ExplorationVessel.js

```diff
@@ -0,0 +1,28 @@
+const Card = require('../../Card.js');
+
+class ExplorationVessel extends Card {
+    //Action: Exhaust up to 3 friendly creatures. Draw 1 card for each house represented among these creatures.
+    //This card has been translated from Chinese and is subject to change.
+    setupCardAbilities(ability) {
+        this.action({
+            target: {
+                mode: 'upTo',
+                numCards: '3',
+                cardType: 'creature',
+                controller: 'self',
+                cardCondition: (card) => !card.exhausted,
```

Comment:
You can, in theory, exhaust friendly exhausted creatures. The effect, at the end, would be the same, though